documentation/polymer-templates/tutorial-template-and-binder.asciidoc

```diff
@@ -10,13 +10,13 @@ layout: page
 
 In this section, we demonstrate how to create a fully-functional template-binder view.
 
-== Creating the Template Component
+It is recommended to use LitElement template language for your components, but PolymerTemplate (see <<tutorial-polymer-template-and-binder#,PolymerTemplate, Combining Templates and Binders>>) is also supported.
```

Comment:
I would not add this at all as the support is deprecated.
IMO it should go the other way around and polymer template docs should start with a disclaimer that it is recommended to use lit instead.